features/conversation-list/conversation-list-item/conversation-list-item-dm.tsx

```diff
@@ -48,11 +53,11 @@ export const ConversationListItemDm = memo(function ConversationListItemDm({
   }, [avatarUri, preferredName, theme]);
 
   const onPress = useCallback(() => {
-    prefetchConversationMessages(currentAccount, topic);
+    prefetchConversationMessages(currentAccount, conversationTopic);
```

Comment:
not as part of this pr, but we should have messages for conversations loaded earlier than this...